### PR TITLE
Refactor REMOTE_STRESS to be possible to run it against the running chain with deployed bridge

### PR DESCRIPTION
**Pull request by dzmitryhil**
* Simplify the REMOTE_STRESS test to produce more load and sped less tokens
* Add new instructions on how to build it and run it locally 
* Refactor contract deployer to let it deploy ERC20 contracts only
* Refactor ERC20 test contracts to mint for sender instead of a constant address

### CODE REVIEW

**Review by AaronKutch — APPROVED** [review]

File: solidity/test/deployERC20.ts [hunk 1]
```diff
@@ -1,166 +1,162 @@
...
-  });
+describe("deployERC20 tests", function () {
 
-  // Duplicate validators and already sorted must fail
-  it("throws MalformedNewValidatorSet on duplicate, sorted validators", async function () {
-    await expect(runTest({ duplicateValidator: true, sortValidators: true })).to.be.revertedWith(
-      "MalformedNewValidatorSet()"
-    );
-  });
+    // Non-duplicate & sorted validators must work
```
Comment (AaronKutch) [inline solidity/test/deployERC20.ts:137]: do we need 4 tests starting here? Don't we just test for being sorted first then test for duplications?

File: tests/REMOTE_STRESS.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
+# Run the remote stress test on the running chain
+
+
+* Export the minter private key for the test
+
+```
+export MINER_PRIVATE_KEY="eth-private-key" # the addres used as fauset for test, usasly we need about
```
Comment (AaronKutch) [inline tests/REMOTE_STRESS.md:7]: multiple typos here

File: solidity/test/deployERC20.ts (hunk 1 above)
Comment (dzmitryhil) [reply solidity/test/deployERC20.ts:137]: The duplicate validation in the solidity contract requires the list to be sorted (costs optimization ), that's why we have 4 tests. 2 with Sorted and 2 without sorting.

File: tests/REMOTE_STRESS.md (hunk 2 above)
Comment (dzmitryhil) [reply tests/REMOTE_STRESS.md:7]: fixed

**Pull request merged**